lib/deaf_grandma.rb

```diff
@@ -13,15 +13,25 @@ def run!
 
     loop do
       user_input = get_user_input
-      p speak(user_input)
+      keeprunning, response = speak(user_input)
+      p response
+      exit if not keeprunning
```

Comment:
Rubyists tend to prefer `unless` for these. 

Even better, flip the return values and do: `exit if done`.